src/components/landing/Landing.js

```diff
@@ -24,22 +24,27 @@ const Landing = ({ history }) => {
 		playerData: [createNewPlayer(0), createNewPlayer(1)]
 	});
 
-	const updatePlayerData = () =>
+	const updatePlayerData = isGameStart =>
```

Comment:
Either provide default value to isGamestart or add value for this parameter everywhere